test/Microsoft.AspNetCore.SignalR.Tests/EndToEndTests.cs

```diff
@@ -290,7 +290,8 @@ public static IEnumerable<object[]> MessageSizesData
                     logger.LogInformation("Sent message", bytes.Length);
 
                     logger.LogInformation("Receiving message");
-                    var receivedData = await receiveTcs.Task.OrTimeout();
+                    // No timeout here because it can take a while to receive all the bytes
```

Comment:
What's a while? Is this a useful test if it takes more than 1 minute? 5 minutes? 10 minutes?